operator/aws.go

```diff
@@ -21,7 +22,8 @@ import (
 )
 
 const (
-	awsRoleAnnotation = "vault.uw.systems/aws-role"
+	awsRoleAnnotation       = "vault.uw.systems/aws-role"
+	defaultStsTTLAnnotation = "vault.uw.systems/default-sts-ttl"
```

Comment:
default here seems a bit confusing to me. As this is per service account configuration I'd expect to set an sts ttl, but not touch the default value that will apply if I omit. I see the code below, so this is more of a question whether it would be more clear to drop the `default` prefix on that key?